pom.xml

```diff
@@ -200,7 +200,7 @@
     <projectVersion>${project.version}</projectVersion>
     <antlr4.version>4.8-1</antlr4.version>
     <maven.site.plugin.version>3.8.2</maven.site.plugin.version>
-    <maven.spotbugs.plugin.version>3.1.12.2</maven.spotbugs.plugin.version>
+    <maven.spotbugs.plugin.version>4.0.0</maven.spotbugs.plugin.version>
```

Comment:
please remove from commit all not yours changes.

Reply:
Thank you for your response. Done!